### ISSUE TITLE
Restyle the Pale Moon website

### ISSUE THREAD

**Issue by wolfbeast** [MoonchildProductions/UXP#964]
In tandem with #963 the Pale Moon website design should get a restyling to make full use of the new graphics (still being created).
General colors will mostly stay the same but may need some tweaking to fit the new design. Fonts will remain the same.

This also impacts other services that copy the style like the Linux site and the Add-ons site. I'd prefer to publish all of this on a single day and for it to be fully polished before publishing, so there's no immediate rush for any of this; just want people to be aware it will be changing.

- [x] Restyle main website
- [x] Adopt style on Add-ons site
- [x] Adopt style on the Wiki
- [x] Adopt style on Linux site (live)
- [x] Publish everything

**Comment by wolfbeast** [comment]
WIP site: http://www-dev.palemoon.org/

**Comment by JustOff** [comment]
Isn't the header too gloomy?

**Comment by wolfbeast** [comment]
> Isn't the header too gloomy?

I don't see why?

**Comment by JustOff** [comment]
Well, dark colors dominate, the clouds are ominous. That is my impression.

**Comment by wolfbeast** [comment]
It's just a rainstorm :) -- Besides i think the range is a decent reflection of what our project deals with.
The art was commissioned and can't easily be changed after the fact.
![PaleMoon-515](https://user-images.githubusercontent.com/3359132/54428988-9623d600-471f-11e9-8048-c99b11a62a95.jpg)

**Comment by JustOff** [comment]
I rather like rainstorms, yet this one scares me as a site header ;)

**Comment by wolfbeast** [comment]
@mattatobin FYI: I decided to keep the current table-based layout instead of converting it to DIVs because if doing that, I'd have to explicitly start using layout tricks and cell-style CSS anyway to keep the rang-fluidity intact that it currently has (and I'll be damned if I'm going to break the website for older browsers by relying on CSS grid or similar).

**Comment by kn-yami** [comment]
In some circumstances the navigation bar will overlay floating social buttons (the current version of the web-page has the same problem):
![an image of the navigation bar problem](https://user-images.githubusercontent.com/34216515/54434367-5f08f100-472e-11e9-8ea8-35c0c52c3e43.png)

**Comment by mattatobin** [comment]
No.

**Comment by wolfbeast** [comment]
decided against this
I'll tweak it in a more minor way later

**Comment by wolfbeast** [comment]
Right, so it just needs a different approach and not be as art heavy

**Comment by wolfbeast** [comment]
Excuse the rusty design skills coming out of a code-melted brain.
See what you guys think now.

**Comment by JustOff** [comment]
Now the darkness has receded, and the moon shines bright - this is a completely different mood and impression :+1:

**Comment by mattatobin** [comment]
The storm clouds still gather even if it isn't so pronounced.. But we can deal with them as they are not the totality of what we do..

I AM NOT A STORM CLOUD!

**Comment by Lootyhoof** [comment]
Something feels a bit...odd about having the wordmark in the page content like that though, even if only for the homepage. The banner image itself looks great.

Just a thought is all.

**Comment by mattatobin** [comment]
There is no realistic way to have the wordmark in the index banner. We tried all combinations.

**Comment by adisib** [comment]
You could have the banner image cycle, but I don't think I like that idea any better. In general, I'm not a big fan of main informational content taking more than 1/4 of the screen to begin, but it isn't that big of a deal and it is probably fine as is.

**Comment by JustOff** [comment]
![PaleMoon-001](https://user-images.githubusercontent.com/340855/54461047-1d9b3480-4774-11e9-8a26-4c262b42f42c.png)
Is the simplest option looking so bad?

**Comment by mattatobin** [comment]
Moonchild has previously stated he doesn't want the wordmark to overlap the moon in any way.. We already tried alt placements none of it worked out very well at all

**Comment by JustOff** [comment]
Actually, I don't, maybe I missed something. However, if this is so, then there is nothing to argue about :disappointed:

**Comment by wolfbeast** [comment]
> Actually, I don't, maybe I missed something. However, if this is so, then there is nothing to argue about disappointed

https://github.com/MoonchildProductions/UXP/pull/985#issuecomment-469912649

**Comment by wolfbeast** [comment]
The wordmark in this design doubles as a page title. On top, doing it the way i did it allows for the fluidity of the layout since it will reflow the catch phrase below the wordmark at smaller widths, catching two birds with one stone.
It may not be "how everyone else does it" but isn't that exactly what identity and personality is about?

**Comment by JustOff** [comment]
> [#985 (comment)](https://github.com/MoonchildProductions/UXP/pull/985#issuecomment-469912649)

Sorry, I apparently didn't realize that you deny this option fundamentally.

> It may not be "how everyone else does it" but isn't that exactly what identity and personality is about?

Of course you decide, but this design still seems to me quite unbalanced.

**Comment by mattatobin** [comment]
Actually it is very balanced.. The only nit I have is there should be a line above the tagline so that it is MORE balanced and also gets the same effect when reflowed.. BUT that is a minor nit.. not worth putting up a fuss about.. Besides, it is JUST the front page.. the subpage/subsite design is less bold or change-y so it will be fine.

**Comment by wolfbeast** [comment]
> but this design still seems to me quite unbalanced.

Feel free to objectively point out where the balance point is then and how it doesn't meet it.

**Comment by wolfbeast** [comment]
Despite me personally hating how it takes away from the artwork's vibrance by slapping text over the moon and reducing its main focus to a background shape without shine, it looks OK-ish enough to do. I may still change it again later. I just can't agree with any design fully I've seen or come up with and just getting tired at staring at the same nits.

**Comment by wolfbeast** [comment]
Final design is up - Keeping it simple.

**Comment by JustOff** [comment]
This is really better. Sorry I was constantly annoying you with my doubts, but it's more honest than keeping silent when they are.

**Comment by mattatobin** [comment]
@JustOff HA.. You're funny cause I involved my self.

**Comment by Lootyhoof** [comment]
I like this - it's a nice compromise given the situation. As a page title this size works fine, and I've no problems with it.

**Comment by wolfbeast** [comment]
( ( `Moonchild Design` * `Tobin Factor` ) / `Ryan Filter` + `JustOff Fear` / `Tobin Factor` ) * `way too much time` = `Success`

**Comment by JustOff** [comment]
Let me share one more thought while you are in a positive mood:

![2019-03-16_22-03-48](https://user-images.githubusercontent.com/340855/54480984-ac24ba00-4837-11e9-979e-29e8c1ab52f6.png)

Y/N?

**Comment by ibuprophen1** [comment]
@JustOff, that's a very nice layout and I do like it very much! :-) 

I'm not sure if you can/want to do this but, maybe you could have the moon (specifically) the same color/shade of blue as the Icon is. 

Just a thought... :-) 


~Ibuprophen

**Comment by kn-yami** [comment]
If you do want to change the icon you would probably want to do it the other way - change the application icon the match the new artwork.

**Comment by Lootyhoof** [comment]
Centred works for me (@JustOff's design) since it looks better aligned with the other stuff that's centred.

Other than THAT, I can't see anything else I could poke at. ;)

**Comment by mattatobin** [comment]
I'd prefer it not centered cause then it brings back the "unbalanced" issue the image form has. Let's just not tempt fate any further.

**Comment by ibuprophen1** [comment]
@JustOff, the following image is the same but, the moon is just slightly more of a Pale Blue. 

![Pale Moon 2](https://user-images.githubusercontent.com/18060271/54481331-b3cd6a00-4809-11e9-87d1-9a69d6f9edf8.jpg)

Just a thought... :-) 


~Ibuprophen

**Comment by mattatobin** [comment]
Decisions on if the actual Pale Moon logo being changed to the pale version used in the new artwork will be dealt with at a later time.

tl;dr .. stop.

**Comment by ibuprophen1** [comment]
Just my final suggestions @JustOff...  

[Pale Moon Suggestion 1](https://user-images.githubusercontent.com/18060271/54481597-225ff700-480d-11e9-8824-ab2eca27973a.png)
[Pale Moon Suggestion 2](https://user-images.githubusercontent.com/18060271/54481602-2e4bb900-480d-11e9-8967-fdafcef6046c.png)
[Pale Moon Suggestion 3](https://user-images.githubusercontent.com/18060271/54481608-39064e00-480d-11e9-9104-fb399e1346ce.png)
[Pale Moon Suggestion 4](https://user-images.githubusercontent.com/18060271/54481613-43c0e300-480d-11e9-9c50-a43dbd5a327f.png)



~Ibuprophen

**Comment by mattatobin** [comment]
... No.

**Comment by wolfbeast** [comment]
@ibuprophen1 You're being disrespectful to the artist of the artwork.

**Comment by mattatobin** [comment]
As if color changed google images would be superior to the fine artwork commissioned, eh?

**Comment by wolfbeast** [comment]
Updated the original issue desc to include checklist for publication. @mattatobin those three are the main locations we need to tackle, right?

**Comment by mattatobin** [comment]
Wiki

**Comment by wolfbeast** [comment]
Right, right. well - I'll publish the main site when you indicate you're far enough along to push it out the same day.

**Comment by wolfbeast** [comment]
Published and done.
